ext/filter/filter_private.h

```diff
@@ -57,6 +57,8 @@
 
 #define FILTER_FLAG_HOSTNAME               0x100000
 
+#define FILTER_FLAG_EMAIL_RFC6531          0x100000
+
```

Comment:
Well, if those flags don't interfere (and they don't from what i can see), anything should pass. Since FILTER_FLAG_HOSTNAME did reuse the value of another define as well. Probably cleaner would be to have those always as next power of 2. On the other hand - there are already over 20 flags, so it slowly moves towards limits on 32-bit for adding further flags. IMHO this value isn't principal right now.

Thanks.